libs/jsonforms-components/src/lib/Controls/FormStepper/FormStepperControl.tsx

```diff
@@ -70,10 +70,25 @@ export const FormStepper = ({
 }: CategorizationStepperLayoutRendererProps) => {
   const uiSchema = uischema as unknown as GoAFormStepperSchemaProps;
   const [step, setStep] = usePersistentState(0);
+  const [isFormValid, setIsFormValid] = useState(false);
   const categories = useMemo(
     () => uiSchema.elements.filter((category) => isVisible(category, data, undefined, ajv)),
     [uiSchema, data, ajv]
   );
+  const handleSubmit = () => {
+    console.log('submitted', data);
+  };
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const vslidateFormData = (formData: Array<UISchemaElement>) => {
+    const validate = ajv.compile(schema);
+    return validate(formData);
+  };
+
+  useEffect(() => {
+    const valid = vslidateFormData(data);
+    setIsFormValid(valid);
```

Comment:
Hi Rama, we can leave the form validation to the future work. We shall not use the ajv compile directly here. Do we have the errors attribute in the props. We can use whether the 'errors' is empty or not to determine the form validation. If this is not the case, we can leave it to the future work.

Reply:
Discussed with @solittlework merging this and trying other way to validate data